### PR TITLE
Migration

### PR DESCRIPTION
**Pull request by JunTaoLuo**
Need to check the removed dependency and code changes.

### CODE REVIEW

File: src/Microsoft.AspNetCore.Hosting/Internal/WebHost.cs [hunk 1]
```diff
@@ -208,15 +208,9 @@ private RequestDelegate BuildApplication()
                 var model = new ErrorPageModel();
                 var runtimeType = Microsoft.Extensions.Internal.RuntimeEnvironment.RuntimeType;
                 model.RuntimeDisplayName = (runtimeType == "CoreCLR") ? ".NET Core" : runtimeType == "CLR" ? ".NET Framework" : "Mono";
```
Comment (JunTaoLuo) [inline src/Microsoft.AspNetCore.Hosting/Internal/WebHost.cs]: Potentially won't need this anymore?

**Comment by pranavkm** [comment]
@Tratcher - did you make the `IAppDiscriminator` change as yet?

**Comment by Tratcher** [comment]
@pranavkm no, this PR can't be merged without breaking DP. I'll work on that after Http.Sys

**Comment by JunTaoLuo** [comment]
Can't merge this until DataProtection's dependency is resolved.

**Comment by pranavkm** [comment]
:shipit:

**Pull request closed without merging**